openprocurement/auction/texas/utils.py

```diff
@@ -102,6 +102,7 @@ def open_bidders_name(auction_document, bids_information):
                         'ru': bids_information[stage['bidder_id']]["tenderers"][0]["name"],
                     }
                 })
+                auction_document[field][index]['bidNumber'] = bids_information[stage['bidder_id']].get('bidNumber', '')
```

Comment:
Можливо додати поле `bidNumber` в межах виклику методу `.update()`?

```suggestion
auction_document[field][index].update({
                    "label": {
                        'uk': bids_information[stage['bidder_id']]["tenderers"][0]["name"],
                        'en': bids_information[stage['bidder_id']]["tenderers"][0]["name"],
                        'ru': bids_information[stage['bidder_id']]["tenderers"][0]["name"],
                    },
                    "bidNumber": bids_information[stage['bidder_id']].get('bidNumber', '')
                })
```

Reply:
До речі, чому не використати `bids_mapping` з контексту як значення за замовчуванням? Там зберігаю усі `binNumber` отримані з `datasource`, якщо вони там були, або згенеровані на ходу.